axelrod/tests/unit/test_moran.py

```diff
@@ -151,30 +261,15 @@ def test_three_players_with_mutation(self):
     def test_four_players(self):
         players = [axelrod.Cooperator() for _ in range(3)]
         players.append(axelrod.Defector())
-        random.seed(10)
+        axelrod.seed(29)
         mp = MoranProcess(players)
         populations = mp.play()
         self.assertEqual(len(mp), 9)
         self.assertEqual(len(populations), 9)
         self.assertEqual(populations, mp.populations)
         self.assertEqual(mp.winning_strategy_name, str(axelrod.Defector()))
 
-    def test_standard_fixation(self):
```

Comment:
It was returning a constant score (fitness) in a 2:1 ratio to see if the outcome was in the ballpark of what would be expected for an r=2 Moran process. This appears to be what the new test essentially does.